ddtrace/llmobs/_utils.py

```diff
@@ -224,6 +224,17 @@ def add_span_link(span: Span, span_id: str, trace_id: str, from_io: str, to_io:
     span._set_ctx_item(SPAN_LINKS, current_span_links)
 
 
+def enforce_message_role(messages: List[Dict[str, str]]) -> List[Dict[str, str]]:
+    """
+    Enforce the message role for the LLM span event.
+    A message with a role of empty content will render as "Message" in the UI.
+    """
+    for message in messages:
+        if message.get("role") is None:
```

Comment:
Is this just to ensure there exists a role field?

Reply:
yeah - will modify to `message.setdefault("role", "")` for a bit more simplicity

Reply:
was there any other concern with this block here?

Reply:
Nope!